File: src/main/java/skyblockclient/commands/SkyblockClientCommands.java
package skyblockclient.commands;

import net.minecraft.client.entity.EntityPlayerSP;
import net.minecraft.command.CommandBase;
import net.minecraft.command.ICommandSender;
import net.minecraft.util.ChatComponentText;
import skyblockclient.SkyblockClient;

public class SkyblockClientCommands extends CommandBase {

    @Override
    public String getCommandName() {
        return "sbclient";
    }

    @Override
    public String getCommandUsage(ICommandSender sender) {
        return "/" + getCommandName();
    }

    @Override
    public int getRequiredPermissionLevel() {
        return 0;
    }

    @Override
    public void processCommand(ICommandSender sender, String[] args) {
        if (args.length == 0) {
            SkyblockClient.display = SkyblockClient.config.gui();
            return;
        }
        EntityPlayerSP player = (EntityPlayerSP) sender;
        String subcommand = args[0].toLowerCase();
        if (subcommand.equals("mimicmessage")) {
            args[0] = "";
            SkyblockClient.config.mimicMessage = String.join(" ", args).trim();
            player.addChatMessage(new ChatComponentText("§aMimic message changed to §f" + String.join(" ", args).trim()));
        }
    }

}

File: src/main/kotlin/skyblockclient/command/SkyblockClientCommands.kt
package skyblockclient.command

import net.minecraft.command.CommandBase
import net.minecraft.command.ICommandSender
import skyblockclient.SkyblockClient.Companion.config
import skyblockclient.SkyblockClient.Companion.display
import skyblockclient.config.Config.mimicMessage
import skyblockclient.utils.Utils.modMessage

class SkyblockClientCommands : CommandBase() {
    override fun getCommandName(): String {
        return "sbclient"
    }

    override fun getCommandAliases(): List<String> {
        return listOf(
            "skyblockclient",
            "sbc"
        )
    }

    override fun getCommandUsage(sender: ICommandSender): String {
        return "/$commandName"
    }

    override fun getRequiredPermissionLevel(): Int {
        return 0
    }

    override fun processCommand(sender: ICommandSender, args: Array<String>) {
        if (args.isEmpty()) {
            display = config.gui()
            return
        }
        val subcommand = args[0].lowercase()
        if (subcommand == "mimicmessage") {
            args[0] = ""
            val message = args.joinToString(" ").trim()
            mimicMessage = message
            modMessage("§aMimic message changed to §f$message")
        }
    }
}